lib/pulse/lib/Pulse.Lib.FlippableInv.fsti

```diff
@@ -27,5 +27,20 @@ val mk_finv (p:slprop { is_storable p }) : stt (finv p) emp (fun x -> off x)
 
 val iname_of #p (f : finv p) : erased iname
 
-val flip_on  (#p:slprop) (fi : finv p) : stt_atomic unit (add_inv emp_inames (iname_of fi)) (off fi ** p) (fun () -> on fi)
-val flip_off (#p:slprop) (fi : finv p) : stt_atomic unit (add_inv emp_inames (iname_of fi)) (on fi) (fun () -> off fi ** p)
+```pulse
+atomic
+val fn flip_on (#p:slprop) (fi : finv p)
+  requires off fi ** p
+  ensures on fi
+  opens [reveal (iname_of fi)]
```

Comment:
I think this is expected behavior. This usually happens with polymorphic instantiation---you need the coercion explicitly.